### ISSUE TITLE
Empty abi produces weird output 

### ISSUE THREAD

**Issue by robcxyz** [sudoblockio/icon-tracker-frontend#290]
Per title, no ABI (ie = []) gives odd output in contracts tab 

https://tracker.icon.community/contract/cxccf103a44f57bd17bc5da486c51a6405b99d29b8#contract

**Comment by FidelVe** [comment]
for this issue the frontend doesnt break, it catches the error and shows the generic output of the error raised from the RPC call.

in this case we can catch that specific error and show a customized message indicating that the contract doesnt have an ABI, what do you think?

**Comment by robcxyz** [comment]
Yeah it should just say that. Just catch ABIs with `len(...) == 0` or something like that with a message

This is rare so not a big deal
